comp/welcome.yaml

```diff
@@ -1,6 +1,6 @@
 summary: Plan the welcome talk for competitors and guests
 
-priority: major
+priority: should
```

Comment:
I think this should stay as a "must". In most cases, it includes fire and other safety information. In the event where the "should" items end up closer to "could", having some kind of intro talk for everyone is going to be even more useful to divulge information.